app/views/shared/check_answers/_no_link_item.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="govuk-summary-list__row" id="app-check-your-answers__<%= name %>">
+  <div class="govuk-summary-list__value" style="width: 90%; word-break: normal">
```

Comment:
Can we make that modification via SASS? Add a class 'no-word-break':
```sass
.normal-word-break {
  .govuk-summary-list__value {
    width: 90%;
    word-break: normal;
  }
  .govuk-summary-list__value {
    text-align: right;
    padding-right: 1px;
    word-break: normal
  }
}
```
Then here the code becomes:
```erb
<div class="govuk-summary-list__row normal-word-break" id="app-check-your-answers__<%= name %>">
  <div class="govuk-summary-list__value">
    <%= question %>
  </div>
  <div class="govuk-summary-list__value">
    <%= answer.present? ? answer : '-' %>
  </div>
</div>
```

Reply:
👍 I had wondered about that myself. I'll change it now.

Reply:
I've taken a slightly different approach to your suggestion and broken the styling down into a number of smaller classes (one for the word-break and several for the various alignment/width/padding combinations) as I couldn't find a one-size-fits-all approach that worked.  No more inline styling though!